mango-core/src/main/scala/org/bdgenomics/mango/models/AlignmentRecordMaterialization.scala

```diff
@@ -161,7 +161,8 @@ object AlignmentRecordMaterialization {
   def loadFromBam(sc: SparkContext, region: ReferenceRegion, fp: String): AlignmentRecordRDD = {
     AlignmentTimers.loadBAMData.time {
       val idxFile: File = new File(fp + ".bai")
-      if (idxFile.exists()) {
+      val idxFile2: File = new File(fp.dropRight(4) + ".bai")
+      if (idxFile.exists() || idxFile2.exists) {
```

Comment:
I would rather just call loadIndexedBam() without a check for bai and handle the exception if no file is found